File: ehe/ehe.Web/Modules/DosyaUrunler/UrunEkleme/UrunSallaColumns.cs
using Serenity.ComponentModel;
using System.ComponentModel;

namespace ehe.DosyaUrunler.Columns;

[ColumnsScript("DosyaUrunler.UrunEkleme")]
[BasedOnRow(typeof(UrunEklemeRow), CheckNames = true)]
public class UrunSallaColumns //urun yeni grid burdan yapiliyo
{
    [EditLink]
    public int UrunlerId { get; set; }
    [EditLink]
    public string UrunAdi { get; set; }
    [EditLink]
    public string UrunAciklamasi { get; set; }
    [EditLink]
    public int UrunStokAdeti { get; set; }
    [EditLink]
    public int UrunFiyati { get; set; }
    [EditLink]
    public int UrunKodu { get; set; }



}

File: ehe/ehe.Web/Modules/ServerTypes/DosyaUrunler/UrunSallaColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { UrunEklemeRow } from "./UrunEklemeRow";

export interface UrunSallaColumns {
    UrunlerId: Column<UrunEklemeRow>;
    UrunAdi: Column<UrunEklemeRow>;
    UrunAciklamasi: Column<UrunEklemeRow>;
    UrunStokAdeti: Column<UrunEklemeRow>;
    UrunFiyati: Column<UrunEklemeRow>;
    UrunKodu: Column<UrunEklemeRow>;
}

export class UrunSallaColumns extends ColumnsBase<UrunEklemeRow> {
    static readonly columnsKey = 'DosyaUrunler.UrunEkleme';
    static readonly Fields = fieldsProxy<UrunSallaColumns>();
}